.travis.yml

```diff
@@ -6,14 +6,18 @@ rvm:
   - 2.3.4
   - 2.2.7
   - jruby
+bundler_args: ''
```

Comment:
Also it was using `install` before and not `bundler_args`. The first one makes caching impossible when this one is supposed to preserve caching.